all.json

```diff
@@ -14,6 +14,48 @@
 		"1inch-network.net",
 		"1inch-wallet.org",
 		"1inch.tk",
+		"accessgains.uk",
+		"acquiomfinances.com",
+		"alpha-aerotek.com",
+		"armetatrade.com",
+		"bittradingenterprises.com",
+		"blue-chip.ltd",
+		"capital-option.co",
+		"capitaltrusts.ltd",
+		"century-trades.com",
+		"champagnecapitals.ltd",
+		"crypto-booster.biz",
+		"cryptotrustfx.live",
+		"expertreliable.co",
+		"globalpower.ltd",
+		"goldenbridgeassets.com",
+		"otc.eqibank.com",
+		"safeedgetrade.com",
```

Comment:
Same comment as above applies to this group. Have no idea how to evaluate these.

Reply:
pictures of the team are always stock photos, etc

Reply:
![image](https://user-images.githubusercontent.com/49607867/166194560-556b56d9-9663-4b5c-a233-192ee6563f7b.png)